crates/uv-cli/src/lib.rs

```diff
@@ -610,14 +610,30 @@ pub struct VersionArgs {
     pub python: Option<Maybe<String>>,
 }
 
-#[derive(Debug, Copy, Clone, PartialEq, clap::ValueEnum)]
+// Note that the ordering of the variants is significant, as when given a list of operations
+// to perform, we sort them and apply them in order, so users don't have to think too hard about it.
+#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, clap::ValueEnum)]
 pub enum VersionBump {
-    /// Increase the major version (1.2.3 => 2.0.0)
+    /// Increase the major version (1.2.3b4.post5.dev6 => 2.0.0)
     Major,
-    /// Increase the minor version (1.2.3 => 1.3.0)
+    /// Increase the minor version (1.2.3b4.post5.dev6 => 1.3.0)
     Minor,
-    /// Increase the patch version (1.2.3 => 1.2.4)
+    /// Increase the patch version (1.2.3b4.post5.dev6 => 1.2.4)
     Patch,
+    /// Make the version stable (1.2.3b4.post5.dev6+local => 1.2.3+local)
+    ///
+    /// This intentionally clears `post` and preserves `local`
```

Comment:
> The choice to clear post here is a bit odd, in that 1.2.3.post4 => 1.2.3 is actually a version decrease, but I think this gives a more intuitive model (as preserving post5 in the previous example is definitely wrong), and also post-releases are extremely obscure so probably no one will notice. 

Clearing post would be a negative bump, imo using stable with a post release should error.